From 8992df8f3a0f5fc16ec41ad6dd7a5a13e6f94d32 Mon Sep 17 00:00:00 2001
From: Vernon Mauery <vernon.mauery@linux.intel.com>
Date: Wed, 5 Dec 2018 16:13:15 -0800
Subject: [PATCH] Add support for 128MB Macronix spi flash MX66L1G45G

This will enable u-boot support for the Macronix MX66L1G45G part.

Change-Id: I5edc69357a8b1607c5c44e53bed3eddf38fdc0be
Signed-off-by: Vernon Mauery <vernon.mauery@linux.intel.com>
---
 drivers/mtd/spi/sf_params.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi/sf_params.c b/drivers/mtd/spi/sf_params.c
index c577d9ed6c..d2a96efe48 100644
--- a/drivers/mtd/spi/sf_params.c
+++ b/drivers/mtd/spi/sf_params.c
@@ -50,6 +50,7 @@ const struct spi_flash_params spi_flash_params_table[] = {
 	{"MX25L12805",	   0xc22018, 0x0,	64 * 1024,   256, RD_FULL,		     WR_QPP},
 	{"MX25L25635F",	   0xc22019, 0x0,	64 * 1024,   512, RD_FULL,		     WR_QPP},
 	{"MX25L51235F",	   0xc2201a, 0x0,	64 * 1024,  1024, RD_FULL,		     WR_QPP},
+	{"MX66L1G45G",	   0xc2201b, 0x0,	64 * 1024,  2048, RD_FULL,		     SECT_4K|WR_QPP},
 	{"MX25L12855E",	   0xc22618, 0x0,	64 * 1024,   256, RD_FULL,		     WR_QPP},
 #endif
 #ifdef CONFIG_SPI_FLASH_SPANSION	/* SPANSION */
-- 
2.17.1

